src/System.Net.Utilities/tests/FunctionalTests/TestSettings.cs

```diff
@@ -10,37 +10,41 @@ namespace System.Net.Utilities.Tests
     internal static class TestSettings
     {
         public static readonly string LocalHost = "localhost";
-        public const int PingTimeout = 200;
+        public const int PingTimeout = 1000;
 
         public const string PayloadAsString = "'Post hoc ergo propter hoc'. 'After it, therefore because of it'. It means one thing follows the other, therefore it was caused by the other. But it's not always true. In fact it's hardly ever true.";
         public static readonly byte[] PayloadAsBytes = Encoding.UTF8.GetBytes(TestSettings.PayloadAsString);
 
         public static Task<IPAddress> GetLocalIPAddress()
         {
-            return ResolveHost(LocalHost, AddressFamily);
+            return ResolveHost(LocalHost);
         }
 
-        public static AddressFamily AddressFamily
-        {
-            get
-            {
-                return AddressFamily.InterNetworkV6;
-            }
-        }
-
-        private static async Task<IPAddress> ResolveHost(string host, AddressFamily family)
+        private static async Task<IPAddress> ResolveHost(string host)
         {
             IPHostEntry hostEntry = await Dns.GetHostEntryAsync(host);
+            IPAddress ret = null;
 
             foreach (IPAddress address in hostEntry.AddressList)
             {
-                if (address.AddressFamily == family)
+                if (address.AddressFamily == AddressFamily.InterNetworkV6)
                 {
-                    return address;
+                    ret = address;
```

Comment:
...and now I see how this was being used. Why not just get rid of this method altogether and inline it into `GetLocalIPAddress`?